### PR TITLE
ClimateMappedAfrica Fix Zoom Bug

### PR DESCRIPTION
**Pull request by kelvinkipruto**
## Description
This PR fixes a bug in ClimateMapped where the map zoom level was not included in the `rootGeography` configuration

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by koechkevin — COMMENTED** [review]
How did this get lost @kelvinkipruto ?

**Comment by kelvinkipruto** [comment]
@koechkevin PR #991 deleted the `RootGeography` file where these changes had been made. When merging #996, I did not crosscheck that further modifications had been made to that file.

**Review by koechkevin — APPROVED** [review]

**Comment by kilemensi** [comment]
If it's a fix, don't wait for my approval, lets get it deployed @kelvinkipruto

**Pull request merged**
